src/chat.rs

```diff
@@ -2334,7 +2421,7 @@ pub(crate) async fn add_contact_to_chat_ex(
         }
         add_to_chat_contacts_table(context, chat_id, contact_id).await?;
     }
-    if chat.param.get_int(Param::Unpromoted).unwrap_or_default() == 0 {
+    if chat.typ == Chattype::Group && chat.is_promoted() {
```

Comment:
I actually like that WA shows a system message "XY added to the list" when someone is added, I guess that's useful to see when someone was added/removed (esp. if they were added before I sent a specific message). It also gives some feedback about the action I just took (add a recipient). I never used broadcasts though, so I don't know _how_ useful it is.

This would obviously require two more stock strings ("added" and "deleted to the list"), so maybe it's not worth it, but I do think it would improve UX.

Reply:
agree, that might be useful.

however, in contrast to groups, as the recipient list is completely controlled by a single person, so it is probably not that important, at least not for the initial implementation. i don't know if whatsapp adds these messages.

Reply:
> i don't know if whatsapp adds these messages

Yes it does add them. But it's fine if we don't add them (now).

Reply:
k, thanks for checking.

i also like the initial messages as "you created a group/broadcast with 123 members.", so maybe we do an info-improvement pr at some point. maybe that fits into the pr "non-blocking group QR joins" #2508 where these info-messages also become much more important.